scripts/windows/readme.md

```diff
@@ -12,23 +12,34 @@ Please use option -h or --help for further reference on how to change the defaul
 
 * CMake (Windows installer can be downloaded from: https://cmake.org/download/)
 
+## Clone Repo
+
+Choose the branch as needed. In our example below we are using the branch/tag v6.0.0:
+* --branch v6.0.0
+
+```console
+git clone --branch v6.0.0 https://github.com/analogdevicesinc/ToF
+cd ToF
+git submodule update --init --recursive
+```
+
 ## Usage
 Open Windows PowerShell then cd to the location of the script. For example:
 ```console
-cd C:\Users\user\ToF\scripts\windows
+cd scripts\windows
 ```
 Get the help menu for more details on how to use the script:
 ```console
-.\setup_project.bat --help
+setup_project.bat --help
 ```
 Run the script with default parameters:
 ```console
-.\setup_project.bat
+setup_project.bat
 ```
 
 Run the script but specify different custom settings. Also, by skipping the '-y' parameter, the script will give a short summary of what actions will do and ask permision to continue or to stop.
 ```console
-.\setup_project.bat -b myBuild -d myDeps -i myInstalledDeps -g "Visual Studio 16 2019" -c Release
+setup_project.bat -b myBuild -d myDeps -i myInstalledDeps -g "Visual Studio 16 2019" -c Release
```

Comment:
myDeps and myInstalledDeps were removed from the scripts argument list in a previous pr